docs/sovereign/software-dependencies.md

```diff
@@ -15,14 +33,28 @@ To install this dependency on Linux, use the following command:
 pip install multiversx-sdk
 ```
 
-### Linux Screen
+### tmux
+
+To install tmux, use the following command:
+```
+sudo apt install tmux
```

Comment:
This should not be mandatory actually... We also have tmux as option in configs.

Reply:
We should put this line in an if and explain what tmux is used for

Reply:
added a note